mlir/lib/Dialect/Tosa/Utils/QuantUtils.cpp

```diff
@@ -107,6 +107,45 @@ void mlir::tosa::computeMultiplierAndShift(double scale, int32_t &multiplier,
   }
 }
 
+// Return a const value for array of int8 vec
+Value mlir::tosa::getConstTensorInt8(OpBuilder &builder, Location loc,
+                                     ArrayRef<int8_t> vec) {
+  int64_t count = vec.size();
+  assert(count > 0);
```

Comment:
nit: could we add a message to these asserts using `... && "<reason>"`

Reply:
Done